### DISCUSSION TITLE
Data re the relevance of another person's vaccination status

### DISCUSSION THREAD

**Discussion by tibbons2** [microCOVID/microCOVID#1008] [Q&A] [answered]
Hi. :-) I'm a new user of microCOVID. And _soooo_ appreciative of the app.

I have a question related to an imminent event. So hoping someone might have some time to help me. I'm hopeful that it would merely be a matter of sharing a link. 

My question concerns the part of the calculator where users can specify another other attendee's vaccination status. We’ve noticed that, all else being equal, according to the calculator, an unvaccinated attendee can increase the risk to us (as fully vaccinated people) by between 5.5 and 6 times. We have suspected such a significant risk, based on various news reports. But we have yet to be able to put our hands on any hard data supporting this. I skimmed through the entire white paper and couldn’t spot any there. But hopefully I am just missing something? Or there is some other data source someone would be willing to share via link?

Thanks so much if so. :-)

**Comment by blanchardjeremy** [comment]
You can read about how we calculate this number here: https://www.microcovid.org/paper/14-research-sources#others-vaccines

We have a "very rough approximation" approach to calculating the difference between "average local resident" and "average vaccinated" and "average unvaccinated." As you can read in #997, we  are considering if that approach is still the best for us to use.  As you can see in #997, our approximate can be off by quite a bit depending on the data. We just don't have "on the ground" numbers of case rate among vaxxed/unvaxxed for all places, so we are currently calculating it based on vaccination rate.

**Reply by blanchardjeremy** [reply]
Oh also, the comment above is about trying to model an "average person who is vaccinated." Which is tricky, because their behavior could have changed after being vaccinated to cancel out some/all of the reductions in risk that the vaccine provides.

If you want to know the underlying data of the efficacy of vaccines given someone whose behavior you know, we have all kinds of info on that: https://www.microcovid.org/paper/14-research-sources#vaccines

**Comment by tibbons2** [comment]
Thanks so much for your reply. :-) Everything you linked was so interesting and I appreciate so much you taking the time.

As I read, I realized that perhaps my question is both more fundamental and easier to answer. My question is, what makes an unvaccinated person, all else being equal (their risk behavior, etc.), more risky for me to be around, even as a vaccinated person. I'm guessing the answer is, simply, because they are more likely to contract Covid. And therefore, they are more likely to be in an infectious state when around me. And, therefore, I am more likely to contract Covid from them, than if they were the same person, but vaccinated. Does that make sense? And, if so, is the data source I'm looking for simply data showing that unvaccinated people are, all else being equal, more likely to contract Covid? And, if so, would you be willing/able to link what microCOVID has on that, or recommend something else good but concise and data intensive answering this, more obvious, question? 

Thanks so much if so. :-)

**Reply by blanchardjeremy** [reply]
Hi there. Yes, the way vaccines work, if an unvaccinated person and a vaccinated person so the exact same activities, the vaccinated person is much less likely to contact COVID. 

I'm not sure what to link you to. We have a lot of scientific papers linked above. Are you looking for something that is digestible? Maybe @beshaya or @mattbell or @jy2wong  has something.

**Reply by tibbons2** [reply]
Sorry I'm just now getting back to you. 

To clarify further, so it's because of this obvious fact about how vaccines work that the calculator works the way it does, when we specify another attendee's vaccination status? That is, because an unvaccinated person is more likely to contract COVID (all else being equal), they're more likely to be infectious when around me, hence my calculator risk goes up having them there, as an unvaccinated v. vaccinated person (all else being equal)? Is it just that simple?

If so, I'm looking for something that spells that out, in a data driven way. Ideally I'd have something quite meaty (for science-interested friends) and something a _little_ more accessible, for others. I'm looking for this because, in my circle, there seems to be skepticism about they idea that an unvaccinated person could pose any real risk to a vaccinated one. Yet I've felt confident it is so, for the reason above. But I'd like to be able to cite something, other than just the calculator results, which usually show 5.5-6x higher risk, when I plug in a vaccinated status for me and an unvaccinated status for the other attendee (and keep all other inputs the same).

I hope that makes sense and that you might have one or two sources like this that you could link. In any event, thank you so much for all your time on this.

**Reply by blanchardjeremy** [reply]
So I think the more relevant point that you're friends are asking about is "can a vaccinated person get COVID."

It's obvious that an unvaccinated person is more likely to get COVID. You can just Google "COVID vaccine effectiveness" and see million articles. 

The real question your friends are asking it seems is: can a person who is vaccinated get COVID. 

To answer that, @beshaya  could probably site better research than I could. But here's an article one of our contributors wrote. 

https://www.mattbell.us/delta-and-long-covid/

It's about delta. But it's also about vaccine effectiveness. If you scroll down to the graph labeled "vaccine effectiveness" you'll see a graph that indicates that vaccinated people A) can still get COVID and B) they are more likely to get infected with COVID  4-6 months after their second dose.

Does that help?

**Comment by beshaya** [comment, marked as answer]
If the question is "is an unvaccinated person necessarily a danger to a
vaccinated person", the answer is no. An unvaccinated person who has been
in hard quarantine (close contact with zero people in the last two weeks)
poses almost exactly zero risk to anyone. If they're willing to be
forthcoming with their activities over this time period, you can use
microCOVID to estimate how much of a danger they are to you. It is entirely
possible for a particular unvaccinated person to be safer than a particular
vaccinated person, or even the average vaccinated person in the area. In
general, I am skeptical of people who refuse to get vaccines taking the
pandemic seriously, so I'm less inclined to trust them to be taking
necessary precautions. But I do recognize that there are people who are
terrified of both COVID and vaccines and may be managing their COVID risk
effectively without getting a vaccine.

**Reply by blanchardjeremy** [reply]
Yep. And people who can't get vaccinated for medical reasons. 👍

**Comment by tibbons2** [comment]
Thanks so much to both of you. Sorry for the long delay in replying. This was all helpful information, especially the long Covid blog post. 

To wrap things up, hopefully, does this about summarize what you both think:

In terms of my question about how the calculator works, putting it all together, I'm thinking any person (including a vaccinated person) is more at risk from being around someone who's unvaccinated (except if they've been in hard quarantine), all else (including that other person's risk behavior, mask wearing, our distance from each other, etc.) being equal, and that this is reflected in the calculator results I see when I change only that factor in the other person (i.e., vaccinated v. unvaccinated). I'm thinking it's just that simple. But would you agree?

And, data-wise, this just requires a showing that 1) unvaccinated people (other than those in hard quarantine), all else being equal (including that other person's risk behavior, mask wearing, distance from me, etc.), are more likely to have/transmit Covid; and 2) vaccinated people _can_ contract Covid. Again, at this point I'm thinking it's just that simple. But would you agree?

Thanks again. :-)
